sleap/gui/commands.py

```diff
@@ -935,11 +975,16 @@ def do_action(cls, context: CommandContext, params: dict):
     @staticmethod
     def ask(context: CommandContext, params: dict) -> bool:
         default_name = context.state["filename"]
-        if default_name:
-            default_name = get_new_version_filename(default_name)
+        if "extension" in params:
+            default_name += f".{params['extension']}"
+            filters = [f"(*.{params['extension']})"]
```

Comment:
Maybe we can pass in an adaptor class in `params["extensions"]` and then use `params["extensions"].default_ext` for the suffix and `params["extensions"].all_ext` for the filters. Also, in that case, `"extensions"` -> `"adaptor"`.